tests/integration/defs/test_e2e.py

```diff
@@ -2677,4 +2679,43 @@ def test_ptp_quickstart_advanced_llama_multi_nodes(llm_root, llm_venv,
     check_call(" ".join(run_cmd), shell=True, env=llm_venv._new_env)
 
 
-# End of Pivot-To-Python examples
+@pytest.mark.timeout(5400)
+@pytest.mark.skip_less_device_memory(80000)
```

Comment:
This marker will check all the gpu's memory, is it safe to set it as 80G, or would you like to set it as 4*80000